### PR TITLE
Add Gin CORS middleware

### PR DESCRIPTION
**Pull request by kaitoo1**
This PR adds and configures Gin's CORS middleware so that we can try and get the frontend working with backend on dev
https://github.com/gin-contrib/cors

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -18,6 +19,10 @@ func Init(pPortInt int,
 
 	pRuntime.Router = gin.Default()
 
+	config := cors.DefaultConfig()
+	config.AllowOrigins = []string{"https://gallery-dev.vercel.app/"}
```
Comment (fbrcsftnr) [inline server/server.go:23]: should we make this an env var?

Comment (fbrcsftnr) [reply server/server.go:23]: also i'm wondering if the default will handle pre-flight requests

Comment (fbrcsftnr) [reply server/server.go:23]: (i think it will)

Comment (kaitoo1) [reply server/server.go:23]: good call on env var - I was thinking just test it out first, and once it works I'll refactor to put things in the right place

**Review by bennycio — APPROVED** [review]

**Comment by kaitoo1** [comment]
Closed in favor of #76

**Pull request closed without merging**